### PR TITLE
[trafficserver] initial integration

### PR DESCRIPTION
**Pull request by 0x34d**
Hello oss-fuzz Team,
FuzzHTTP is not deployed right now due to broken compile system against coverage and undefined sanitizer.

bug:https://github.com/apache/trafficserver/issues/9158

Signed-off-by: 0x34d <ajsinghyadav00@gmail.com>

### CODE REVIEW

File: projects/trafficserver/project.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+homepage: "https://trafficserver.apache.org"
+language: c++
+primary_contact: "security@apache.org"
```
Comment (jonathanmetzman) [inline projects/trafficserver/project.yaml:3]: Do you have permission from upstream? Do they know they wont be able to see any bugs with this?

**Comment by jonathanmetzman** [comment]
We approve of adding this project.

**Comment by 0x34d** [comment]
@bryancall call?

File: projects/trafficserver/project.yaml (hunk 1 above)
Comment (bryancall) [reply projects/trafficserver/project.yaml:3]: If you would like to send security alerts for the Apache Traffic Server project, you can send emails to security@trafficserver.apache.org

File: projects/trafficserver/project.yaml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 homepage: "https://trafficserver.apache.org"
 language: c++
 primary_contact: "security@apache.org"
+vendor_ccs:
+  - "security@trafficserver.apache.org"
```
Comment (bryancall) [inline projects/trafficserver/project.yaml:5]: 👍🏻 looks good

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**